### ISSUE TITLE
NOD/NVG Toggle Off

### ISSUE THREAD

**Issue by willmon8** [DrSuperGood/SC2-UA3#10]
NVG toggle in early game leaves scenery much darker than before powered on.  This issue has existed in UA3 for a long time (forever?).  Not a high priority.

**Comment by willmon8** [comment]
From DSG:

>This is caused by the NVG altering the time of day model used. When it restores the model, it does not correctly restore the time it was at and hence it will look different. One can notice this with the shadows objects cast as they move around. Honestly NVG should probably be removed as it is more annoying than adding to the gameplay. UAC has no NVG as a result.

**Comment by ChildServices** [comment]
"Honestly NVG should probably be removed as it is more annoying than adding to the gameplay. UAC has no NVG as a result."
Whilst I agree that it's annoying, I don't think it should be removed entirely from the game. Instead, make it required for less of the game. Maybe at most the second half of chapter 1 and the first half of chapter 2.

**Comment by willmon8** [comment]
I agree with ChildServices.  UA2 was beautifully dark and spooky.  The lowlight aspects of UA3 add value to the gameplay from an immersion perspective and should be maintained to some extent.  However, needing to spend the whole game with NVG on is also kind of annoying as DSG says.  CAS comes online at 0600, or 30(?) minutes into the storyline, according to the Ravensword dialogue.  Perhaps game could start in darkness and then spend everything after 30 minutes in daylight.  This should also eliminate the need for a fix regarding NVG altering time of day, since there's no way to initially be toggling it off before darkness arrives.  Just a thought, without knowing what the programming is like.

I'd be just as happy leaving this alone the way it has always been and living with the green.

**Comment by DrSuperGood** [comment]
If one wanted NOD/NVG for immersion then one could have tied it to the actual lighting instead of an option for players to turn on/off. It is practically impossible to play with it off due to the low lighting level anyway so there really is not an option to the players. This would also fix the moving shadow problem since players cannot alter lighting on demand.

At the time I could not alter it because of large community pressure. Now that UAC removed it I am sure people will be more understanding of any changes made.

**Comment by ChildServices** [comment]
GreenVision™ is part of UA3's identity, so I don't think it should be entirely gone. Still, I think it should be required for slightly less of the game than it is now. As it stands, you're playing 4/5ths of the game in monochrome green.
